server/hotrod/src/main/resources/META-INF/services/org.infinispan.factories.components.ModuleMetadataFileFinder

```diff
@@ -0,0 +1 @@
+org.infinispan.server.hotrod.ServerMetadataFileFinder
```

Comment:
It's not strictly necessary to pass ```null``` when segment filtering is not desired, there's an overloaded method for that:
```CloseableIterator<Entry<Object, Object>> retrieveEntries(String filterConverterFactory, int batchSize);```